CHANGELOG.rst

```diff
@@ -155,6 +155,7 @@ Interactive improvements
 - ``read --help`` and friends no longer ignore redirections. This fixes a regression in version 3.1 (:issue:`10274`).
 - Measuring a command with ``time`` now considers the time taken for command substitution (:issue:`9100`).
 - ``fish_add_path`` now automatically enables verbose mode when used interactively (in the commandline), in an effort to be clearer about what it does (:issue:`10532`).
+- The `accept-autosuggestion` input function now returns true when there was something to accept.
```

Comment:
Pedantically, I think the real change is that it now returns false if there *wasn't* something to accept.